run_tests.py

```diff
@@ -0,0 +1,3 @@
+import code_detection.tests
+
+code_detection.tests.run()
```

Comment:
I'm envisioning that we might end up adding tests for other stuff eventually 🤷

Reply:
Was about to say to add this into the tests file itself, but yeah we might be adding tests for other things indeed